src/content/en/updates/2019/08/native-file-system.md

```diff
@@ -115,19 +119,19 @@ Once the user selects a file, `chooseFileSystemEntries()` returns a handle,
 in this case a [`FileSystemFileHandle`][fs-file-handle] that contains the
 properties and methods needed to interact with the file.
 
-It’s helpful to keep a reference to the file handle around so that it
-can be used later. It’ll be needed to save changes back to the file, or
-to perform any other file operations. In the next few milestones, installed
-Progressive Web Apps will also be able to save the handle to IndexedDB
-and persist access to the file across page reloads.
+It’s helpful to keep a reference to the file handle around so that it can be
+used later. It’ll be needed to save changes back to the file, or to perform any
+other file operations. In the next few versions of Chrome, installed Progressive
+Web Apps will also be able to save the handle to IndexedDB and persist access to
+the file across page reloads.
 
 #### Read a file from the file system
 
-Now that you have a handle to a file, you can get the properties of the file,
-or access the file itself. For now, let’s simply read the contents of the
-file. Calling `handle.getFile()` returns a [`File`][file-api-spec] object,
-which contains binary data as a blob. To get the data from the blob, call
-one of the reader methods (`slice()`, `stream()`, `text()`, `arrayBuffer()`).
+Now that you have a handle to a file, you can get the file's properties, or
+access the file itself. For now, let’s simply read its contents. Calling
+`handle.getFile()` returns a [`File`][file-api-spec] object, which contains
+binary data as a blob. To get the data from the blob, call one of the reader
```

Comment:
I suggest deleting either 'binary data' or 'as a blob' to eliminate the redundant term.

Reply:
Let's change to 'contains a blob.'

Reply:
Let's go with `as a blob` then.